frontend/src/components/Layout/Header/SearchCourses/SearchResult/SearchResultItem/SearchResultItem.tsx

```diff
@@ -9,17 +8,19 @@ import { LABEL_MESSAGE } from 'constants/messages';
 import { ICourse } from 'types/course';
 import { convertDurationToString } from 'utils/helpers/convertDurationToString';
 
-import { Image, CourseTitle, SearchResultCourse } from './styled';
+import { Image, CourseTitle, SearchResultCourse, RedirectButton } from './styled';
 
 interface CourseFound {
   course: ICourse;
   addDivider: boolean;
   status?: string;
-  redirectTo: string;
+  onClick: (event: React.MouseEvent<HTMLElement>) => void;
 }
 
-const SearchResultItem: React.FC<CourseFound> = ({ course, status, addDivider, redirectTo }) => (
-  <Link key={course._id} to={redirectTo}>
+const LABEL_MESSAGE = 'New';
```

Comment:
already exist in constants/messages
<img width="650" alt="image" src="https://user-images.githubusercontent.com/44285339/163170747-43bee854-a919-4480-8d6c-19a11854dd60.png">